db/migrate/20230509214723_separate_trusted_users_can_edit_others_setting.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class SeparateTrustedUsersCanEditOthersSetting < ActiveRecord::Migration[7.0]
+  def up
+    execute "
+    DO
+    $do$
+    BEGIN
+      IF EXISTS (SELECT * FROM site_settings WHERE name = 'trusted_users_can_edit_others') THEN
+        DELETE FROM site_settings WHERE name = 'trusted_users_can_edit_others';
+        INSERT INTO site_settings (name, data_type, value, created_at, updated_at) VALUES ('edit_all_topic_groups', 20, '', now(), now());
+        INSERT INTO site_settings (name, data_type, value, created_at, updated_at) VALUES ('edit_all_post_groups', 20, '', now(), now());
+      END IF;
+    END
+    $do$
+    "
```

Comment:
No strong feelings about this, but I wonder if we can stick to 

```
execute <<~SQL
IF EXISTS...      
SQL
```

```
execute <<~SQL
INSERT INTO...
SQL
```

mostly because when everything's a string there is no syntax highlighting for the symbols, and also to keep to how we usually write these.